Sources/KeyboardSupport/KeyboardRespondable.swift

```diff
@@ -44,6 +44,12 @@ extension UIViewController {
     }
 }
 
+/// KeyboardScrollable will ask a TextInputView that conforms to this protocol for preferred distance between the field and the keyboard.
```

Comment:
We might want to change "TextInputView" to "UITextInputView" just to be 100% consistent.

Reply:
https://github.com/BottleRocketStudios/iOS-KeyboardSupport/pull/36/commits/7fa3afd9cf3e11ff9feac3cd993b6980a49139d4